docs/source/migration/4.0.mdx

```diff
@@ -154,9 +184,7 @@ Those helper functions:
 * make `CacheFirst`, `NetworkFirst` and `CacheAndNetwork` policies ignore fetch errors.
 * throw ApolloComposite exception if needed.
 
-Because of the number of different options in version 3 and the complexity of error handling, these functions may not 100% match the 3 behavior, especially in the advanced cases involving watchers. If you are in one of those cases, we strongly recommend using the 4 functions that are easier to reason about.
-
-## Other Apollo Runtime changes
+Because of the number of different options in version 3 and the complexity of error handling, these functions may not 100% match the 3 behavior, especially in the advanced cases involving watchers. If you are in one of those cases, we strongly recommend using the version 4 functions that are easier to reason about.
```

Comment:
```suggestion
Because of the number of different options in version 3 and the complexity of error handling, these functions may not 100% match the version 3 behavior, especially in the advanced cases involving watchers. If you are in one of those cases, we strongly recommend using the version 4 functions that are easier to reason about.
```